vita3k/renderer/include/renderer/types.h

```diff
@@ -148,6 +148,11 @@ struct FragmentProgram : ShaderProgram {
 struct VertexProgram : ShaderProgram {
 };
 
+struct ShaderHash {
```

Comment:
std::pair is better. Maybe be better to name ProgramHash

Reply:
i have thinks some diferent, but here is better, pair is good, but only struct can said me what is frag/vert, 
and no, is wrong, here is ShaderHash, no Programhach, and ProgramHash already exist for program hashes so

Reply:
There are hashes of two shader with two types, so better named something else then ShaderHash. I can accept the first one